### ISSUE TITLE
There is no “label for” property for visual instructions

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1916]
URL: http://clarity-ks.surge.sh/radios

Tools: Web Accessibility Toolbar (WAT2.0) / ARIA Markup

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Radios tab.
3. Check the markup in Radios tab.

Actual Results:
There are no labels associating "Radio Buttons" text, “Default/Stacked radio button group” text, “Inline radio button group” text and “These are disabled radio buttons” text.
For more details, please refer to "Radios-NoMarkup.zip".

Expected Results:
Set “label for” property for "Radio Buttons" text, “Default/Stacked radio button group” text, “Inline radio button group” text and “These are disabled radio buttons” text.

Note:
This issue occurs on other tabs, such as Progress tab, Radios tab, Wizards tab.

Violation of Section 508 Standards: 
Section 1194.22 (n) When electronic forms are designed to be completed on-line, the form shall allow people using assistive technology to access the information, field elements, and functionality required for completion and submission of the form, including all directions and cues.

User Impact: Blind.

Screen shot:
1.	Radios-NoMarkup.png

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/7e583ead-e981-4eff-ad3c-6d0a157dc1a0)

2.	LabelForAndID'sMarkup.png

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/ef2566ae-d429-43bc-b2a3-6e6057258f35)

NOTE: this may also be a problem on the website

**Comment by Jinnie** [comment]
As we're not using fieldsets, which is an _option to consider_, I've added  role="radiogroup" aria-labelledby="label_id" to the form-group divs.
It works well with screen readers, the group label is read when changing between radio groups.
